scenarios/centos-9/hive.yml

```diff
@@ -0,0 +1,8 @@
+---
+cifmw_hive_platform: "openstack"
+cifmw_hive_action: "claim_cluster"
+cifmw_hive_dry_run: false
+cifmw_hive_openstack_pool_name:
+cifmw_hive_namespace:
+claim_name:
+cifmw_hive_kubeconfig:
```

Comment:
How should I pass these?

Reply:
are those private parameters? If so, they should be in zuul's secrets I guess.